src/components/buttons/icon-button/icon-button.percy.js

```diff
@@ -140,5 +140,35 @@ screenshot('IconButton', () => (
         theme="blue"
       />
     </Spec>
+
+    <Spec label="type - when button">
```

Comment:
What would be visually different in these cases? Doesn't seem like we need percy tests for this 🤔 . I would remove them.

Reply:
I agree that it's unlikely that this breaks. However browsers might have default styles for different button types or somebody might confuse `type` for `tone` while working on the component. This is here to ensure that there are none and the buttons look the same - no matter the type. There's no cost to adding more specs (aside from more to review) and we don't need to worry about it again once it's approved. I'd keep them for peace of mind.

That said, buttons will probably be redone soonish anyways so I we don't need to argue about this a lot. If you feel strongly about it, please go ahead and just push onto this branch :)